Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -391,7 +457,7 @@ function Enable-SecureFTPDeploymentForAppServices
     {
         Write-Host "No App Service(s) found with FTP State $($AllAllowed). Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
         Write-Host $([Constants]::DoubleDashLine)
-        if(($appServicesWithFTPStateAllAllowed|Measure-Object).Count -gt 0) 
+        if($AutoRemediation -and ($appServicesWithFTPStateAllAllowed|Measure-Object).Count -gt 0) 
```

Comment:
This condition is wrong, $totalAppServicesWithFTPStateAllAllowed and $appServicesWithFTPStateAllAllowed|Measure-Object are same thing.
You should use total number of app service if greater than zero ($totalAppServices = ($appServiceResources | Measure-Object).Count)

Reply:
1. The logic is different here saurabh. We just wanted to check the AppServices which have FTP deployment as AllAllowed and we just want that count. If we'll use totalAppServices it will be working unnecessarily for all the AppServices.
2.I know these ($totalAppServicesWithFTPStateAllAllowed and $appServicesWithFTPStateAllAllowed|Measure-Object) are the same things but if we run auto remediation only then these resources are getting saved. and I am able to see the log file with remediated and skipped resources.